recipes/jupyter_conda/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jupyter_conda" %}
+{% set version = "2.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fb429fb3328306d434debcba67c33183934c08c45aa238eaf0584ca868428109
+
+build:
+  skip: True  # [py2k]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python =3
```

Comment:
Would change this to `python >=3`

Reply:
I did that at the beginning but I got the following error in appveyor in bfd76bc:
```bash
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\bld\\jupyter_conda_1543259999808\\work_moved_jupyter_conda-2.3.5-py>=3_0_win-64_main_build_loop'
```
It disappears when I switch from `python >=3` to `python =3`. For [ipython](https://github.com/conda-forge/ipython-feedstock/blob/master/recipe/meta.yaml), they only set `skip` for python 2 and no specification for python. Should I use that approach instead?